File: DikUcgenKontrolu.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#python3


def hesapla(*args):
	sirala = sorted(list(args),key=int)
	if (sirala[-1])**2 == (sirala[-2])**2+(sirala[-3])**2:
		return args,"Dik Üçgen."
	else:
		return args,"Dik Üçgen Değil."



kenar1 = int(input("Üçgenin Birinci Kenarını Giriniz: "))
kenar2 = int(input("Üçgenin İkinci Kenarını Giriniz: "))
kenar3 = int(input("üçgenin üçüncü Kenarını Giriniz: "))

print(hesapla(kenar1, kenar2, kenar3))

File: dikucgenkontrolu.rb
def kontrol(a, b, c)


  if((a*a)+(b*b)==(c*c))
    puts "Bu üçgen dik üçgendir."
  else
    puts "Bu üçgen dik değildir."
  end
end

print "Birinci kenarı giriniz:"
a=Integer(gets)
print  "İkinci kenarı giriniz:"
b=Integer(gets)
print  "Üçüncü kenarı giriniz:"
c=Integer(gets)

kontrol(a,b,c)
